RES-GUENTHER/extra/socket/socket-01-unix.c

```diff
@@ -15,52 +19,64 @@ int main(int argc, char *argv[])
 {
     printf("start \n");
     fflush(stdout);
+    // if argument 1 == "server", we are the server
     bool isServer=false;
     if  (argc>1 && strcmp(argv[1],"server")==0){
         isServer=true;
     }
```

Comment:
```suggestion
    bool isServer = argc>1 && strcmp(argv[1],"server")==0;
```

Reply:
If it was hard to write, it should be hard to understand!

Reply:
@Steve2955 Im Prinzip ja, aber das hier ist ja Lernmaterial $\rightarrow$ nicht zielführend